src/components/OurOffer/OfferCard/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Box, Typography, Button } from "@mui/material";
+import classnames from "classnames";
+import CheckIcon from "@assets/icon/check-icon.svg?react";
+import StarIcon from "@assets/icon/star-icon.svg?react";
+import { IPlanProps, ITextProps } from "../helpers";
+import PlanPrice from "./PlanPrice";
+import useStyles from "./styles";
+
+interface OfferCardProps {
+  plan: IPlanProps;
+  text: ITextProps;
+}
+
+const OfferCard: React.FC<OfferCardProps> = ({ plan, text }) => {
+  const { classes } = useStyles();
+
+  function handleFeatureList() {
+    return plan.features.map((feature, index) => (
+      <li
+        key={index}
+        className={classnames({
+          [classes.featureItem]: true,
+          [classes.titleFeature]: feature.includes(text.benefitText),
```

Comment:
The expression `feature.includes(text.benefitText)` appears multiple times. You could outsource it to a variable instead.